upgrade/sql/9.0.1.sql

```diff
@@ -0,0 +1,7 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8mb4';
+
+INSERT INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
+  -- https://github.com/PrestaShop/PrestaShop/pull/39277
+  (NULL, 'actionModifyHtmlPurifierConfig', 'Called when configuring HTMLPurifier', 'Allows modules to modify the HTMLPurifier definition by adding custom allowed HTML elements or attributes during Tools::purifyHTML().', '1')
+ON DUPLICATE KEY UPDATE `title` = VALUES(`title`), `description` = VALUES(`description`);
```

Comment:
```suggestion
ON DUPLICATE KEY UPDATE `title` = VALUES(`title`), `description` = VALUES(`description`);

```

Let's add one new empty line and we're good.

Reply:
@Quetzacoalt91
Done

Thank you!